### ISSUE TITLE
Codelist (V 2.1) - CriteriaTaxonomy - Quality Assurance & Environmental as fourth branch

### ISSUE THREAD

**Issue by JachymHercher** [OP-TED/ESPD-EDM#219]
In the ESPD Regulation, Part IV has a point D - Quality Assurance Schemes and Environmental Management Standards. However, the CriteriaTaxonomy codelist seems to have only three main selection criteria branches (corresponding to points A-C). 

The following three values look like they correspond to point D.

```
CRITERION.SELECTION.TECHNICAL_PROFESSIONAL_ABILITY.CERTIFICATES.QUALITY_ASSURANCE.QA_INSTITUTES_CERTIFICATE
CRITERION.SELECTION.TECHNICAL_PROFESSIONAL_ABILITY.CERTIFICATES.QUALITY_ASSURANCE.QA_INDEPENDENT_CERTIFICATE
CRITERION.SELECTION.TECHNICAL_PROFESSIONAL_ABILITY.CERTIFICATES.ENVIRONMENTAL_MANAGEMENT.ENV_INDEPENDENT_CERTIFICATE

```

Why are these values in the "Technical professional ability" branch and not in a fourth branch of their own?

(FYI @muricna)

**Comment by paulakeen** [comment]
This is the result of a discussion that was never ended properly and that had two aspects:

**Aspect I**: the decision was made that **the ESPD specification SHOULD not refer to (nor be organised according to) the "Parts" into which the ESPD Regulation Annex** is structured (but to the concepts it contains, instead). Thus in the online documentation you will see no mention to any of these "Parts".

**Aspect II:** if you look into the Criteria Taxonomy data structure you will notice that:

1. The Criteria Taxonomy groups criteria by data structure;
2. All three criteria have identical data structures;
3. The descriptions of all three criteria refer to "certificates";
4. The criterion about institutes certificate, in the Regulatation Annex, falls under Part IV.C, point (12) (Technical and professional abilities)

So, we had the dilema of having two tabs for QA certificates (one as one QA abilities-related certificate and one for two exclusive QA certificates) or just all three grouped into one single QA abilities-related sub-branch (as all three share the same data structure).

We chose to have them all under the same sub-branch.

But we can reopen this discussion and do it differently, if you think that is worthwhile.

**Comment by ec-mcs** [comment]
Reading through the comment from Enric, I would suggest to stick to this approach and not to artificially create a D section. As well it would look rather bizarre in the forms why there would be a separate category for environmental.

**Comment by mireiaBM** [comment]
This issue has been solved in version 2.1.0 of the ESPD-EDM
